PostingGuidelines.md

```diff
@@ -0,0 +1,28 @@
+What Should Be Posted
+=====================
+* **DO** post items that would be interesting or useful to the technical / programming community
+* **DO** post original blog posts where you have developed a solution to a problem
+* **DO** post summaries of other blog posts and how they apply to our work - make sure to link to to the source for these
+* **DO** post personal or project achievements if it relates to a particular project or initiative
+* **DO** post links to interesting technologies and articles and explain how we might use them. For these be sure to tag them with "InterestingTech" - Tags coming soon
+* **DO NOT** post internal documentation even if it does not contain sensitive information
+
+
+How To Post
+===========
+* Checkout the latest version of the [repository](https://github.com/emory-libraries/emory-libraries.github.io)
+* Create a new branch for your post
+* Make sure you have an entry for your username in `_data/_authors.yml`
+* Create a file in the `_posts` directory of the format `YYYY-MM-DD-TITLE.md`
+* In the above file, include the below header information. Note the author field links to the reference in `_data/_authors.yml`:
+```
+---
+layout: post
+title: THE TITLE
+categories: [Jekyll]
+author: yang_li
+---
+```
+* Write your post
+*  On GitHub create a [pull request](https://github.com/emory-libraries/emory-libraries.github.io/compare?expand=1) for others to review
```

Comment:
Any guidelines on which kind of posts merit/require review? Or do all of them, even "interesting tech" posts?